src/app/Components/MenuOptions.js

```diff
@@ -0,0 +1,27 @@
+export default function MenuOptions() {
+  const divMenu = document.createElement('div');
+  divMenu.classList.add('container');
+  document.body.append(divMenu);
+  const options = ['People', 'Starships', 'Vehicles'];
+  options.forEach((option) => {
+    let btn = document.createElement('button');
+    btn.setAttribute('data-option', option);
+    btn.classList.add('option');
+    btn.textContent = option;
+    divMenu.append(btn);
+  });
+  const menuOption = {
+    selectOption: '',
+  };
+  const menuElements = document.querySelectorAll('.option');
+  function handleSelectOption() {
+    menuOption.selectOption = this.dataset.option;
+    menuElements.forEach((element) => {
+      element.classList.remove('select_option');
```

Comment:
Please look at [BEM class naming convention](http://getbem.com/naming/) and tweak these here